tardis/montecarlo/tests/test_packet_source.py

```diff
@@ -44,98 +28,97 @@ def blackbodysimplesource(self, request):
         """
         cls = type(self)
         montecarlo_configuration.LEGACY_MODE_ENABLED = True
-        cls.bb = BlackBodySimpleSource(base_seed=1963, legacy_second_seed=2508)
-        yield cls.bb
+        bb = BlackBodySimpleSource(
+            radius=123,
+            temperature=10000 * u.K,
+            base_seed=1963,
+            legacy_second_seed=2508,
+        )
+        yield bb
         montecarlo_configuration.LEGACY_MODE_ENABLED = False
 
+    def test_bb_nus(self, regression_data, blackbodysimplesource):
+        actual_nus = blackbodysimplesource.create_packet_nus(100).value
+        expected_nus = regression_data.sync_ndarray(actual_nus)
+        assert_allclose(actual_nus, expected_nus)
+
+    def test_bb_mus(self, regression_data, blackbodysimplesource):
+        actual_mus = blackbodysimplesource.create_packet_mus(100)
+        expected_mus = regression_data.sync_ndarray(actual_mus)
+        assert_allclose(actual_mus, expected_mus)
+
+    def test_bb_energies(self, regression_data, blackbodysimplesource):
+        actual_unif_energies = blackbodysimplesource.create_packet_energies(
+            100
+        ).value
+        expected_unif_energies = regression_data.sync_ndarray(
+            actual_unif_energies
+        )
+        assert_allclose(actual_unif_energies, expected_unif_energies)
+
+    def test_bb_attributes(self, regression_data, blackbodysimplesource):
+        actual_bb = blackbodysimplesource
+        expected_bb = regression_data.sync_hdf_store(actual_bb)[
+            "/black_body_simple_source/scalars"
+        ]
+        assert_allclose(expected_bb.base_seed, actual_bb.base_seed)
+        assert_allclose(expected_bb.temperature, actual_bb.temperature.value)
+        assert_allclose(expected_bb.radius, actual_bb.radius)
+
+
+class TestBlackBodySimpleSourceRel:
     @pytest.fixture(scope="class")
-    def blackbody_simplesource_relativistic(self, request):
+    def blackbody_simplesource_relativistic(self):
         """
-        Create BlackBodySimpleSourceRelativistic instance.
+        Create BlackBodySimpleSource instance.
```

Comment:
This should still be relativistic, right?

Reply:
Thanks Andrew, I fixed this in https://github.com/tardis-sn/tardis/pull/2584/commits/9983c82267757fd1fe11f97eae9a16b0d5bde6f1